### PR TITLE
remove Plugins from Singleton get_state, fix ray launcher default key pair name

### PR DESCRIPTION
**Pull request by jieru-hu**
remove Plugins from Singleton get_state (#1312)
fix key pair name - add user (#1309)

### CODE REVIEW

**Comment by omry** [comment]
Are you planning on releasing hydra 1.0.6 for this?
if so, you should include a news fragment.

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+### Features
+
+### API Change (Renames, deprecations and removals)
+
+### Bug Fixes
+
+### Plugins
+
+### Maintenance Changes
+- Remove `Plugins` from `Singleton.get_state()` ([#1330](https://github.com/facebookresearch/hydra/issues/1330))
```
Comment (omry) [inline NEWS.md:13]: This file is generated from the news fragments, do not edit it.

**Review by omry — CHANGES_REQUESTED** [review]

**Comment by jieru-hu** [comment]
> Are you planning on releasing hydra 1.0.6 for this?
> if so, you should include a news fragment.

ah I see, i checked https://github.com/facebookresearch/hydra/tree/1.0_branch/news before this and didnt see anything under there so I thought 1.0_branch behaves differently. will fix.

**Comment by jieru-hu** [comment]
> > Are you planning on releasing hydra 1.0.6 for this?
> > if so, you should include a news fragment.
> 
> ah I see, i checked https://github.com/facebookresearch/hydra/tree/1.0_branch/news before this and didnt see anything under there so I thought 1.0_branch behaves differently. will fix.

nvm, should have checked here https://github.com/facebookresearch/hydra/blob/a8daf4823fb4b155f55fa9ada13394423e8cbfed/website/versioned_docs/version-1.0/development/documentation.md

**Comment by omry** [comment]
> > Are you planning on releasing hydra 1.0.6 for this?
> > if so, you should include a news fragment.
> 
> ah I see, i checked https://github.com/facebookresearch/hydra/tree/1.0_branch/news before this and didnt see anything under there so I thought 1.0_branch behaves differently. will fix.

Release branch is supposed to be stable, so no news is good news :)

**Review by omry — APPROVED** [review]
Updated comments a bit.
feel free to merge once CI passes.

**Pull request merged**